### DISCUSSION TITLE
EffectorP 3.0?

### DISCUSSION THREAD

**Discussion by bob100077** [ccdmb/predector#72] [General]
Are there any plans to introduce EffectorP 3.0 - fungi to the pipeline? 
I know this version is fairly recent.

Thanks!
Chris Bonner

PS. Thank you so much for putting this together, it's been a total pain working between them, especially with dependencies and installations.

Sperschneider J and Dodds PN (2021) EffectorP 3.0: prediction of apoplastic and cytoplasmic effectors in fungi and oomycetes. MPMI. Abstract

**Comment by darcyabjones** [comment]
Hi Chris!

Yes it's always been harder than it should be.
I hope the pipeline helps, but of course I only really hear from people when things go wrong so it's hard to gauge :).

The latest versions of the pipeline do run EffectorP 3 already.
It isn't incorporated into the ML ranking method yet (i'm currently working on an update to include it) but the results are in the tables.

I haven't included the "fungi" mode though if that's what you mean.
From what I remember reading there wasn't really any benefit to excluding the oomycete model when classifying fungal effectors.
I'll benchmark the "fungi" mode and see whether I should include it as a separate column as I've done for deepredeff.


Happy to hear suggestions or thoughts :)

**Reply by bob100077** [reply]
Great!
Again, thank you so much for all of your hard work putting these into a user friendly pipeline!
